include/ebpf_nethooks.h

```diff
@@ -161,6 +161,8 @@ typedef enum
 
 /**
  * @brief Get current pid and tgid (sock_addr specific only).
+ * This helper function is there only to support legacy programs.
+ * New sock_addr programs should use bpf_get_current_pid_tgid helper instead.
```

Comment:
```suggestion
 *
 * @deprecated Use bpf_get_current_pid_tgid instead.
```

Use the `@deprecated` doxygen markup.